lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -569,7 +569,7 @@ def get_msg_type(parameters)
 
         initiator =
           case parameters[:stored_credential][:initiator]
-          when 'customer' then 'C'
+          when 'cardholder' then 'C'
```

Comment:
i don't know how likely it is, but in case someone is using 'customer' in this field, wonder if we should do:
`when 'cardholder', 'customer' then 'C'`
to keep it backwards compatible

Reply:
I agree.

Reply:
Great point. Thanks for the reviews @britth and @curiousepic! This has been updated.